cirrus/server/tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+from pytest import fixture
+from fastapi.testclient import TestClient
+from cirrus.main import app
+
+
+@fixture(scope="module")
+def client():
+    with TestClient(app) as c:
+        yield c
```

Comment:
Curious to know why you are using a yield here

Reply:
oh, the reason is that the yield statement returns the client object to any functions that call it. Once pytest finishes running all the tests, it also automatically closes the TestClient context/cleanups resource when it's ready to exit from this module.